### PR TITLE
Add the support of HRNet-W48

### PR DESCRIPTION
**Pull request by PkuRainBow**

### CODE REVIEW

File: MODEL_ZOO.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 We use the [TorchVision](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py)
 ResNet implementation, which is not exactly the same as the 
 [TensorFlow](https://github.com/tensorflow/models/blob/master/research/deeplab/core/resnet_v1_beta.py) 
 ResNet implementation.
+We use the [HRNet-Semantic-Segmentation](https://github.com/HRNet/HRNet-Semantic-Segmentation) HRNet implementation.
+
+### Cityscapes baselines (Under Progress)
+
+All the results are under progress and we are making efforts to finetune various hyperparameters to achieve better performance. 
+
```
Comment (bowenc0221) [inline MODEL_ZOO.md:14]: Probably revert this change?

File: MODEL_ZOO.md [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+We use Pytorch1.5 in all of our experiments. The docker is [rainbowsecret/pytorch1.5:latest](https://hub.docker.com/repository/docker/rainbowsecret/pytorch1.5).
```
Comment (bowenc0221) [inline MODEL_ZOO.md:15]: Maybe use a separate table for ResNet and HRNet backbone? You can associate this with the HRNet table, and we can add result for HRNet18 and 32 in the future.

File: configs/panoptic_deeplab_H48_os4_cityscapes.yaml [hunk 3]
```diff
@@ -2,28 +2,29 @@ MODEL:
...
-      LOW_LEVEL_CHANNELS_PROJECT: (32, 16)
+      LOW_LEVEL_CHANNELS_PROJECT: (64, 32, 16)
       DECODER_CHANNELS: 128
       HEAD_CHANNELS: 32
       ASPP_CHANNELS: 256
       NUM_CLASSES: (1, 2)
       CLASS_KEY: ["center", "offset"]
 DATASET:
-  ROOT: "./datasets/cityscapes"
+  ROOT: "../../../../dataset/original_cityscapes"
```
Comment (bowenc0221) [inline configs/panoptic_deeplab_H48_os4_cityscapes.yaml:27]: Please use the default path `./datasets/cityscapes`.

File: configs/panoptic_deeplab_H48_os4_cityscapes.yaml [hunk 4]
```diff
@@ -74,6 +75,6 @@ POST_PROCESSING:
   NMS_KERNEL: 7
   TOP_K_INSTANCE: 200
   STUFF_AREA: 2048
-OUTPUT_DIR: "./output/panoptic_deeplab_R50_os16_cityscapes"
+OUTPUT_DIR: "./output/panoptic_deeplab_H48_B8_O4_90K_cityscapes"
 GPUS: (0, 1, 2, 3)
```
Comment (bowenc0221) [inline configs/panoptic_deeplab_H48_os4_cityscapes.yaml:79]: Are you training with 4 GPUs or 8 GPUs?

File: segmentation/model/backbone/hrnet_config.py [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+HRNET_18.STAGE4.NUM_CHANNELS = [18, 36, 72, 144]
+HRNET_18.STAGE4.BLOCK = 'BASIC'
+HRNET_18.STAGE4.FUSE_METHOD = 'SUM'
+
+
+MODEL_CONFIGS = {
+    'hrnet18': HRNET_18,
+    'hrnet32': HRNET_32,
+    'hrnet48': HRNET_48,
+}
```
Comment (bowenc0221) [inline segmentation/model/backbone/hrnet_config.py:130]: Can you put this in the [segmentation/config](https://github.com/bowenc0221/panoptic-deeplab/tree/master/segmentation/config) folder? I feel it is better to group config together.

File: MODEL_ZOO.md (hunk 1 above)
Comment (PkuRainBow) [reply MODEL_ZOO.md:14]: fixed

File: MODEL_ZOO.md (hunk 2 above)
Comment (PkuRainBow) [reply MODEL_ZOO.md:15]: fixed
